apps/judicial-system/web/src/routes/Court/Indictments/Overview/Overview.tsx

```diff
@@ -79,16 +79,13 @@ const IndictmentOverview = () => {
         <PageTitle>{formatMessage(strings.inProgressTitle)}</PageTitle>
         <CourtCaseInfo workingCase={workingCase} />
         {workingCase.defendants?.map((defendant) =>
-          defendant.subpoenas?.map(
-            (subpoena) =>
-              subpoena.subpoenaId && (
-                <ServiceAnnouncement
-                  key={`${subpoena.id}-${subpoena.created}`}
-                  subpoena={subpoena}
-                  defendantName={defendant.name}
-                />
-              ),
-          ),
+          defendant.subpoenas?.map((subpoena) => (
+            <ServiceAnnouncement
```

Comment:
The reason for the subpoenaId check was to make sure it had been successfully sent to the police before displaying the service status, is that being handled elsewhere now?

Reply:
I noticed that, but there is no reason not to show the subpoena service status even if the system is in the porcess of sending the subpoena to the police. Sending the subpoena is handling in a queue and could take some time. The handling of null police subpoena ids is now in the use subpoena hook.